pkg/storage/local.go

```diff
@@ -1510,6 +1510,80 @@ func (is *ImageStoreLocal) copyBlob(repo, blobPath, dstRecord string) (int64, er
 	return -1, zerr.ErrBlobNotFound
 }
 
+// blobStream is using to serve blob range requests.
+type blobStream struct {
+	reader io.Reader
+	closer io.Closer
+}
+
+func NewBlobStream(blobPath string, from, to int64) (io.ReadCloser, error) {
```

Comment:
Just a thought, if we're not going to add this as part of the interface (and I see signature is different between s3 and local), maybe we should not export it?

Reply:
Yes, this method is internal to the implementations, we can un-import. Fixed.